client/src/components/RatingsAndReviews/Review.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import ReviewStars from '../common/ReviewStars'
 
 const Review = ({review}) => {
   return (
     <div className="rr-review-container">
       <div className="rr-header">
-        <h2>---Hello from Individual Review---</h2>
-        <div>***** Rating: {review.rating}</div>
+        <div>
+          <ReviewStars rating={review.rating}/>
```

Comment:
Can you provide some unique identifier here like the review id and pass it to ReviewStars with the ratingId prop. This is so the hidden radio inputs and labels in the ReviewStars will be uniquely identified for accessibility reasons.

Reply:
<ReviewStars rating={review.rating} ratingId={review.review_id}/>

would that work?

Reply:
@squeakypickles33 yes that would